src/sysmem.h

```diff
@@ -32,7 +32,16 @@
 **  This is used in the function 'SyAllocBags' below.
 */
 extern Int SyStorMax;
-extern Int SyStorOverrun;
+
+// SyStorOverrun is used for reporting to the user if more than SyStorMax
+// memory has been allocated. It is used as there is more than once place
+// this can be reported to the use.
```

Comment:
I don't understand the final sentence. Actually, re-reading, the first sentence also is a bit odd... SyStorOverrun is not reporting to the user itself after all. How about something like this instead:
> SyStorOverrun tracks whether an allocation exceeding the memory limit specified by
> SyStorMax is exceeded. Several places in the GAP code base check its values and
> will take appropriate actions if it is set to a value different from SY_STOR_OVERRUN_CLEAR.